scrapy/commands/__init__.py

```diff
@@ -15,7 +17,7 @@ class ScrapyCommand:
     crawler_process = None
 
     # default settings to be used for this command instead of global defaults
-    default_settings = {}
+    default_settings: Dict[str, Any] = {}
```

Comment:
Should keys be str? We support callables as keys in settings now.

Reply:
Do you mean after #3873? Those are values, keys can be other objects but only as sub-keys within other settings (`ITEM_PIPELINES`, `SPIDER_MIDDLEWARES`, etc)